pkgs/applications/networking/sync/openrsync/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openrsync";
+  version = "unstable-2022-05-08";
+
+  src = fetchFromGitHub {
+    owner = "kristapsdz";
+    repo = "openrsync";
+    rev = "f50d0f8204ea18306a0c29c6ae850292ea826995";
+    hash = "sha256-4tygoCQGIM0wqLfdWp55/oOPhD3lPUuTd9/LXQAASXU=";
+  };
+
+  # Uses oconfigure
+  prefixKey = "PREFIX=";
+
+  meta = with lib; {
+    homepage = "https://www.openrsync.org/";
+    description = "BSD-licensed implementation of rsync";
+    license = licenses.isc;
+    maintainers = with maintainers; [ fgaz ];
+    # https://github.com/kristapsdz/openrsync#portability
+    # https://github.com/kristapsdz/oconfigure#readme
+    platforms = with platforms; linux ++ freebsd ++ netbsd ++ openbsd;
```

The utimensat issue is x86_64-darwin only since the SDK was too old. We can also test if `darwin.apple_sdk_11_0.callPackage` helps.

```suggestion
    platforms = platforms.unix;
    broken = stdenv.isDarwin && stdenv.isx86_64;
```

---

`darwin.apple_sdk_11_0.callPackage` fixed it